src/_N001_Email.ino

```diff
@@ -108,10 +110,24 @@ boolean NPlugin_001_send(const NotificationSettingsStruct& notificationsettings,
 			mailheader.replace(String(F("$emailfrom")), address);
 		}
 
-		mailheader.replace(String(F("$nodename")), Settings.Name);
-		mailheader.replace(String(F("$emailfrom")), notificationsettings.Sender);
+		//mailheader.replace(String(F("$nodename")), Settings.Name);
+		//mailheader.replace(String(F("$emailfrom")), notificationsettings.Sender);
```

Comment:
Why disable these replacements? If you don't want them, just take them out of the header